tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,81 @@
+"""Task 1 and 2 of Module 2."""
+
+from collections import defaultdict
+# create CONSTANT variables for weight
+LIGHT_WEIGHT_THRESHOLD = 60
+HEAVY_WEIGHT_THRESHOLD = 70
+MAX_INVENTORY_CAPACITY = 80
+
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+# update dragonLoot variable changed to dragon_loot
+dragon_loot = [
+
```

Comment:
This extra space is not necessary, could you scan the code with pylint/flake8 and try to address the styling issues?